containers/proxy-httpd-image/Dockerfile

```diff
@@ -46,6 +46,4 @@ RUN sysconf_addword /etc/sysconfig/apache2 APACHE_MODULES proxy_http
 COPY uyuni-configure.py /usr/bin/uyuni-configure.py
 RUN chmod +x /usr/bin/uyuni-configure.py
 
-COPY ssl.conf /etc/apache2/vhosts.d/ssl.conf
```

Comment:
If so, then I think we can drop completely the `ssl.conf` file in the `containers/proxy-httpd-image` folder, right?

Reply:
Indeed, I forgot to remove it. Will do